Package.swift

```diff
@@ -23,8 +23,7 @@ let package = Package(
             targets: ["DittoDiskUsage"]),
     ],
     dependencies: [
-        // Ditto.diskUsage was added in 3.0.1
-        .package(url: "https://github.com/getditto/DittoSwiftPackage", from: "3.0.1"),
+        .package(url: "https://github.com/getditto/DittoSwiftPackage", from: "4.0.0-beta1"),
```

Comment:
v4 is out. We should change this.

```suggestion
        .package(url: "https://github.com/getditto/DittoSwiftPackage", from: "4.0.0"),
```